demo_drivers/meshing/adaptive_tet_meshes/tetmesh_faceted_surfaces.h

```diff
@@ -506,13 +506,12 @@ class DiskTetMeshFacetedSurface : public virtual TetMeshFacetedSurface
        for (unsigned j=0;j<3;j++)
         {
          Node* nod_pt=el_pt->node_pt(j);
-         if (nod_pt->is_on_boundary()) count++;
+         if (nod_pt->is_on_boundary(b)) count++;
         }
        if (count==2)
         {
          is_on_boundary[el_pt]=true;
-         if ((el_pt->node_pt(0)->is_on_boundary())&&
-             (el_pt->node_pt(1)->is_on_boundary()))
+         if (!el_pt->node_pt(2)->is_on_boundary(b))
```

Comment:
I'd put brackets around the thing that's negated, i.e. if (!(el_pt....))

Reply:
Added in the latest commit, see the URL below for the changes applied.
https://github.com/oomph-lib/oomph-lib/pull/97/commits/6309f360261807c16f6d5b80452be2205c85237b